SQLDataSource.test.js

```diff
@@ -32,32 +31,6 @@ describe("Configuration", () => {
 
     expect(initialize).toThrow(Error);
   });
-
-  test("an warning to be logged if batching is used", () => {
```

Comment:
why remove these tests?

Reply:
because remove raw queries and make batching like in apollo-datasource-rest. It`s show better performance for big count of queries